src/hartreefock.jl/operator.jl

```diff
@@ -493,7 +494,8 @@ function Calc_Expec(binfo,Chan1b,Chan2b,HFobj,Op_Rp2,dict_2b_ch,dict6j,MatOp,to;
         nthre = nthreads()
         keychs = [ zeros(Int64,3) for i=1:nthre]
         S912s = [ zeros(Float64,nthre) for i=1:2]
-        @inbounds @qthreads for idx_a =1:length(allps)
```

Comment:
Just a quick comment (you do NOT need to take into account for JOSS), best practices for Julia is to use `for index in eachindex(allps)` rather than `for index in 1:length(allps)` since it allows arrays with arbitrary indices. Similarly, `pairs` is preferred over `enumerate`.